src/game.zig

```diff
@@ -1010,6 +1111,10 @@ pub fn update(deltaTime: f64) void { // MARK: update()
 			move[i] = a/frictionCoefficient*deltaTime - c_1/frictionCoefficient*@exp(-frictionCoefficient*deltaTime) + c_1/frictionCoefficient;
 		}
 
+		if(Player.super.climbing and Player.crouching and move[2] < 0) {
+			move[2] = 0;
```

Comment:
This is bad since it allows you to negate any fall damage by jumping onto a ladder. There should be a velocity limit beyond which you cannot hold onto the climbable block.